<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentDlg</class>
 <widget class="QDialog" name="AppointmentDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>640</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Outpatient">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>881</width>
     <height>381</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>门诊号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>门诊名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>值班医生工号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>接诊日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>接诊时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>容量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>门诊地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>门诊联系电话</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_ManageOutPatient">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>640</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>管理门诊</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_Appointment">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Modern No. 20</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预约门诊：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_PatientName">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_PatientName">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>就诊人</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_OutpatientName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_OutpatientName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预约科室</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Time">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>580</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>上午</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>下午</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_Time">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>580</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>就诊时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Date">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>就诊日期</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Date">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>570</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>9</month>
     <day>10</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>9</month>
     <day>10</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_Image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>69</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Outpatient.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AppointmentDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppointmentDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
